<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>470</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #fcf0ca;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_done">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
border-style: outset;
border-width: 2px;
border-radius: 10px;
background-color: #bde0ff;
font: 75 bold 18pt &quot;FreeSerif&quot;;

</string>
    </property>
    <property name="text">
     <string>Вперёд</string>
    </property>
   </widget>
   <widget class="QLabel" name="design_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Здравствуй, пользователь!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="design_label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Камень, ножницы, бумага — популярная игра на руках, известная во многих странах мира.

Игра была изобретена в Китае. Военачальники эпохи поздней династии Хань играли в игру, называемую шоушилин. Также существует японский вариант игры «дзян-кэн» или «индзян». Сыграем?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="design_label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Выбери уровень сложности:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
    <property name="singleStep">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="design_label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>320</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;FreeSerif&quot;;</string>
    </property>
    <property name="text">
     <string>Выбери количество игр:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
